packages/text/src/Text.ts

```diff
@@ -10,27 +10,6 @@ import type { IDestroyOptions } from '@pixi/display';
 import type { ICanvas, ICanvasRenderingContext2D } from '@pixi/settings';
 import type { ITextStyle } from './TextStyle';
 
-// The type for Intl.Segmenter is only available since TypeScript 4.7.2, so let's make a polyfill for it.
```

Comment:
Looks like all the ISegmenter stuff got removed?

Reply:
oh that strange...